### DISCUSSION TITLE
Having a concussion removes slow penalty for diving suits

### DISCUSSION THREAD

**Discussion by fernandeez** [FakeFishGames/Barotrauma#14070] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

i posted it on reddit and someone told me to post it here
having a concussion removes the slow debuff of diving suits(also makes you run funny, kinda like officer earl from cloudy with a side of meatballs)
someone also tested it and said on reddit it only works on diving suits
someone also said that being drunk might also remove the debuff because it also has a different animation

### Reproduction steps

1. start a game 2. get a concussion 3. run

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
no crashes
```

**Comment by itchyOwl** [comment]
Thanks for the report, created a ticket: #14081.

**Comment by Regalis11** [comment]
The issue has now been fixed in our private repo, coming up in a future update.

**Reply by fernandeez** [reply]
https://tenor.com/view/siuuu-gif-24393397
